app/models/application_record.rb

```diff
@@ -4,6 +4,9 @@ class ApplicationRecord < ActiveRecord::Base
   class_attribute :schema_validations_loaded
```

Comment:
this file is copied via the installer but I like this better, just need to remove it from the installer